### PR TITLE
Add changelog for v3 and few previous releases

### PR DESCRIPTION
**Pull request by nikitin-da**

### CODE REVIEW

**Comment by nikitin-da** [comment]
Closes #859

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,99 @@
...
+* [PR 858](https://github.com/pushtorefresh/storio/pull/858): Rename package to storio3.
+* [PR 861](https://github.com/pushtorefresh/storio/pull/861): Fix maven url, update version.
+* [PR 862](https://github.com/pushtorefresh/storio/pull/862): Android gradle plugin 3.0.1.
+* [PR 864](https://github.com/pushtorefresh/storio/pull/864): Add interceptors for ContentResolver.
+* [PR 865](https://github.com/pushtorefresh/storio/pull/865): Add gradle versions plugin.
+* [PR 866](https://github.com/pushtorefresh/storio/pull/866): Mockito 2.13.0 and Mockito-Kotlin 1.5.0.
+* [PR 867](https://github.com/pushtorefresh/storio/pull/867): Kotlin 1.2.0.
+* [PR 870](https://github.com/pushtorefresh/storio/pull/870): Do not publish jar for android modules.
+
+## Version 2.1.0
```
Comment (geralt-encore) [inline CHANGELOG.md]: how did old changelog end here?

Comment (nikitin-da) [reply CHANGELOG.md]: `Changelog.md` was not updated we just set tag's description =(

Comment (geralt-encore) [reply CHANGELOG.md]: sadpanda.jpg

**Comment by artem-zinnatullin** [comment]
We should just leave a link to releases, keeping changelog under version
control is a bit ridiculous :)

**Comment by nikitin-da** [comment]
@artem-zinnatullin but we have ability to review. It's especially important if it contains migration notes ;)

**Pull request merged**